bibtex/bib/quantum.bib

```diff
@@ -2210,6 +2328,16 @@ @Article{WTH+11
   Url                      = {http://arxiv.org/abs/1105.1165}
 }
 
+@Article{WZ82,
+  Title                    = {A single quantum cannot be cloned},
+  Author                   = {Wootters, William K and Zurek, Wojciech H},
+  Year                     = {1982},
+
+  Owner                    = {albraodb},
+  Publisher                = {Nature Publishing Group},
+  Timestamp                = {2015.06.05}
+}
```

Comment:
Yeah, that was a bad reference (no journal name, either), so I re-fetched it. This is an example where the published version does not have the full first names of the authors. I am not sure I should fill them in myself.